### PR TITLE
Chapters: Intro, 02, 03, 04

### PR DESCRIPTION
**Pull request by Danielle9897**
This is the commit from the first 2 days of Dana's work.
The next chapters will be committed  *per chapter*.

### CODE REVIEW

File: Ch02/Ch02.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ setup. After that, you can skip to the next section, where we'll actually start
 The easiest way to get RavenDB is probably via Docker. If you already have Docker installed, all you need to
 do is run the following command (using PowerShell):
 
-    $rvn_args = "--Setup.Mode=None --License.Eula.Accepted=true"
-    docker run 
-      -p 8080:8080 
-      -e RAVEN_ARGS=$rvn_args 
+    $rvn_args = "--Setup.Mode=None --License.Eula.Accepted=true" `
```
Comment (ayende) [inline Ch02/Ch02.md:27]: Shouldn't have a ` in the end

File: Ch02/Ch02.md [hunk 2]
```diff
@@ -230,7 +230,7 @@ update {
 
 Click the `Test` button, and you can see the results of running this operation: a category document. You can also select which
 specific document this test will be tested on. The before-and-after results of running this script on `categories/4-A` are 
-shown in Figure 2.6.
+shown in Figure Figure 2.7.
```
Comment (ayende) [inline Ch02/Ch02.md:233]: Figure Figure

File: Ch02/Ch02.md [hunk 3]
```diff
@@ -274,8 +273,7 @@ a simple console application to explore the RavenDB API. Typically, RavenDB is u
 so we'll also explore some of the common patterns of organizing your RavenDB usage in your application later in 
 this chapter.
 
-The code samples in this book use C#, but the documentation can guide you on how to achieve the same results in any
-supported clients. 
+The code samples in this book use C#, but the documentation can guide you on how to achieve the same results with any supported client.
```
Comment (ayende) [inline Ch02/Ch02.md:276]: Should probably change it to "most of the cost samples"

File: Ch03/Ch03.md [hunk 4]
```diff
@@ -289,7 +288,7 @@ to our kindergarten.
 ```
 
 
-All of the information is in one place. You can see that we used a semantic ID, `children/`, with the child name as the document ID. This 
+All of the information is in one place. For the document ID you can see that we used a semantic ID, children/alice-liddell, which includes the child name. This
```
Comment (ayende) [inline Ch03/Ch03.md:291]: `children/alice-liddell` should be using `

File: Ch03/Ch03.md [hunk 5]
```diff
@@ -336,7 +335,7 @@ reference. Alice and Henry (and Edith) only hold the _identifier_ for their pare
 How is this model reflected in our code? Let's look at Listing 3.4 to see that (again, with some information redacted to make it easier
 to focus on the relevant parts).
 
-```
+```{caption="Child class representing the model" .json}
```
Comment (ayende) [inline Ch03/Ch03.md:338]: This will create a new listing number, no? Where is that refered?

File: Ch02/Ch02.md (hunk 1 above)
Comment (Danielle9897) [reply Ch02/Ch02.md:27]: done

File: Ch02/Ch02.md (hunk 2 above)
Comment (Danielle9897) [reply Ch02/Ch02.md:233]: done

File: Ch02/Ch02.md (hunk 3 above)
Comment (Danielle9897) [reply Ch02/Ch02.md:276]: done

File: Ch03/Ch03.md (hunk 4 above)
Comment (Danielle9897) [reply Ch03/Ch03.md:291]: done

File: Ch03/Ch03.md (hunk 5 above)
Comment (Danielle9897) [reply Ch03/Ch03.md:338]: The Listing number *already existed* (3.4). It was just missing the caption.

**Pull request merged**